libcontainer/rootfs_linux.go

```diff
@@ -776,7 +776,7 @@ func mountPropagate(m *configs.Mount, rootfs string, mountLabel string) error {
 	}
 
 	if err := syscall.Mount(m.Source, dest, m.Device, uintptr(flags), data); err != nil {
-		return err
+		return fmt.Errorf("Dan mountPropagate Failed %s %s %d %s %v", m.Source, dest, flags, data, err)
```

Comment:
Although this is one of my biggest complaints with runc, is the error messages should give us more information about where they happened.  I will fix this, but I think we should make an effort to state things like this to show that mount propagate failed on SRC DEST ... Operation not supported.